nuxeo-services/nuxeo-platform-query-api/src/main/java/org/nuxeo/ecm/platform/query/nxql/NXQLQueryBuilder.java

```diff
@@ -739,7 +739,13 @@ public static Boolean getBooleanValue(DocumentModel model, PredicateFieldDefinit
      * @since 8.4
      */
     public static String appendClause(String query, String clause) {
```

Comment:
nit: could be interesting to have a more generic `appendClauses()` based on a `Stream.of(..).filter(..).collect(..)` solution

Reply:
need a backport and don't want to change API